src/components/TeamProjectCard.vue

```diff
@@ -59,8 +59,7 @@ export default Vue.extend({
   backface-visibility: hidden;
   cursor: pointer;
    &:hover, &:focus {
-      border: 3px solid $rocket;
-      box-shadow: 0px 1px 4px rgba($rocket, 0.4);
+      box-shadow: 0px 15px 8px rgba($white, 0.35);
```

Comment:
There should be 4 px units. x offset, y offset, blur, spread. The 15px was supposed to set the blur but right settings the y offset which doesn't look that great since we want an even shadow effect.

Reply:
Yep my bad 😅 .  looks like this now so more even: 
<img width="449" alt="Screen Shot 2020-04-23 at 11 21 00 AM" src="https://user-images.githubusercontent.com/16490844/80063954-a928ec80-8554-11ea-8224-4acc7ed189ef.png">